app/src/evaluation/literalai_exporter.py

```diff
@@ -61,6 +65,8 @@ def to_csv_dict(self) -> dict[str, str]:
             "Has Chat History": str(self.has_chat_history),
             "Thread ID": self.thread_id,
             "Timestamp": self.timestamp,
+            "Feedback Scores": self.scores[0] if len(self.scores) == 1 else self.scores,
+            "Feedback Comments": self.comments[0] if len(self.comments) == 1 else self.comments,
```

Comment:
Crux of PR. 
Also if there's only one item (99%+ of the time), don't format the items as a list (which uses brackets) -- just encode the single item as a string.